lib/jsonapi/serializer.ex

```diff
@@ -38,6 +38,7 @@ defmodule JSONAPI.Serializer do
       id: view.id(data),
       type: view.type(),
       attributes: view.attributes(data, conn),
+      meta: view.meta(data, conn),
```

Comment:
@jeregrine I coded this based on your comments in the original issue:

> have a default view.ex meta function that returns an empty %{} and is overrideable.

You want to deviate from that now?